bokeh/models/ui/tooltips.py

```diff
@@ -0,0 +1,113 @@
+#-----------------------------------------------------------------------------
```

Comment:
Why is the module name pluralized? Do we anticipate more/different tooltip models in the future?

Reply:
I name all such modules plural, even if there's currently only one model. I've been contemplating splitting off persistent tooltips, but I'm not sure about this.